docs/src/install/install.rst

```diff
@@ -21,7 +21,8 @@ After downloading the Opaque codebase, build and test it as follows.
                    sudo mkdir /opt/spark
                    sudo mv spark-3.1.1*/* /opt/spark
                    rm -rf spark-3.1.1*
-                   sudo chmod -R +wx /opt/spark/work # may not be needed, depends on where Spark is installed
+                   sudo mkdir /opt/spark/work
+                   sudo chmod -R a+wx /opt/spark/work
```

Comment:
Should we change the instructions here to support downloading Spark to another location? I usually download it to ~/

Reply:
When I was looking at install guides, the most common locations I saw were /usr/local and /opt. https://refspecs.linuxfoundation.org/FHS_3.0/fhs-3.0.pdf also mentions that /opt is for add-on applications

Reply:
Ok, then let's keep this!